Add CLI Examples for CloudWatch Internet Monitor

kocanaog opened the pull request:
*Issue #, if available:*

*Description of changes:*

Add CLI Examples for CloudWatch Internet Monitor. Link: https://docs.aws.amazon.com/cli/latest/reference/internetmonitor/#cli-aws-internetmonitor

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

elysahall commented on awscli/examples/internetmonitor/create-monitor.rst:
```diff
@@ -0,0 +1,16 @@
+**To create a new internet monitor**
+
+This example uses the ``create-monitor`` command to create a new internet monitor. ::
```
```suggestion
The following ``create-monitor`` example creates a new internet monitor. ::
```

All your examples need their intro lines updated to follow the format of "The following ``command-name`` example..."
https://aws.github.io/aws-cli/docs_styleguide.html#description

elysahall commented on awscli/examples/internetmonitor/create-monitor.rst:
```diff
@@ -0,0 +1,16 @@
+    {
+        "Arn": "arn:aws:internetmonitor:us-east-1:123456789012:monitor/TestMonitor",
+        "Status": "PENDING"
+    }
```
All your examples need a "For more information..." line with a link.
https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link

elysahall commented on awscli/examples/internetmonitor/delete-monitor.rst:
```diff
@@ -0,0 +1,8 @@
+    aws internetmonitor delete-monitor \
+        --monitor-name TestMonitor
+
+This command returns to the prompt if successful.
```
```suggestion
This command produces no output.
```

elysahall commented on awscli/examples/internetmonitor/stop-query.rst:
```diff
@@ -0,0 +1,9 @@
+        --monitor-name TestMonitor \
+        --query-id AQICAHjE50ADcWtUTXGTlgtIHH4U_xxxxxxxx
+
+This command returns to the prompt if successful.
```
```suggestion
This command produces no output.
```

elysahall commented on awscli/examples/internetmonitor/tag-resource.rst:
```diff
@@ -0,0 +1,9 @@
+        --resource-arn arn:aws:internetmonitor:us-east-1:123456789012:monitor/TestMonitor \
+        --tags Environment=Prod,Type=App
+
+This command returns to the prompt if successful.
```
```suggestion
This command produces no output.
```

elysahall commented on awscli/examples/internetmonitor/untag-resource.rst:
```diff
@@ -0,0 +1,9 @@
+        --resource-arn arn:aws:internetmonitor:us-east-1:123456789012:monitor/TestMonitor \
+        --tag-keys Environment Type
+
+This command returns to the prompt if successful.
```
```suggestion
This command produces no output.
```

elysahall reviewed (changes requested):
Hi @kocanaog,
There were some formatting issues with the examples you submitted. Please fix these issues, commit, and push your changes.

kocanaog commented:
I fixed all the files with the new changes -
* Updated beginning lines to 'The following ``command`` example ~'
* For commands with no output, replaced 'This command returns to the prompt if successful.' with 'This command produces no output.'
* Provided 'For more information' sections in all files.

@elysahall
